README.md

```diff
@@ -45,3 +45,21 @@ For testing
 ```
 npm test
 ```
+
+For lint
+
+```
+npm lint
+```
+
+For e2e
+
+```
+npm run e2e
+```
+
+For run production locally
```

Comment:
> This change should be in PR #18, though.

It will be, this branch was created based on `feature/improve-angularjs-webpack-babel-app`, so after the merge, all changes will also be on the base branch

Reply:
Changed